src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/DotNetSdkHelper.cs

```diff
@@ -93,11 +93,13 @@ public string ExecuteNew(string projectType, string projectName, string projectD
         {
             options += $" --language \"{language}\"";
         }
-        if (string.IsNullOrEmpty(customArgs))
+        if (!string.IsNullOrEmpty(customArgs))
         {
             options += $" {customArgs}";
         }
 
+        //temporary console writeline to keep track of where tests are
+        Console.WriteLine($"new {projectType} {options}", projectDirectory);
```

Comment:
Erroneously included in the PR?

Reply:
Yep, this will be removed before it's checked in. It's useful in my testing atm.

Reply:
This has been removed as of latest commit.